data_import/models.py

```diff
@@ -6,14 +6,10 @@
 from django.core.urlresolvers import reverse
 from django.db import models
```

Comment:
Do we need these models at all after the removal? or is that because of the weird "project data being a subclass" thing that we need to keep it around for now?

Reply:
To respond here - the DataFile model is still used, but as an obsolete non-abstract parent to a child class (which is used for everything now)... and there are no other children. So this could definitely be cleaned up, but since it's nontrivial, not worrying about it here. :)